mmdet3d/ops/pointnet_modules/point_sa_module.py

```diff
@@ -49,12 +47,12 @@ def __init__(self,
                  fps_mod: List[str] = ['D-FPS'],
                  fps_sample_range_list: List[int] = [-1],
                  dilated_group: bool = False,
-                 norm_cfg: dict = dict(type='BN2d'),
                  use_xyz: bool = True,
-                 pool_mod='max',
+                 pool_mod: str = 'max',
                  normalize_xyz: bool = False,
-                 bias='auto'):
-        super().__init__()
+                 grouper_return_grouped_xyz: bool = False,
+                 grouper_return_grouped_idx: bool = False):
```

Comment:
Reformat these input arguments to be consistent with ours?

Reply:
Please also reformat others

Reply:
Done.